packages/rainbowkit/src/wallets/walletConnectors/metaMaskWallet/metaMaskWallet.ts

```diff
@@ -98,13 +46,14 @@ export const metaMaskWallet = ({
       opera: 'https://addons.opera.com/extensions/details/metamask-10',
       browserExtension: 'https://metamask.io/download',
     },
-
-    mobile: {
-      getUri: shouldUseWalletConnect ? getUri : undefined,
-    },
-    qrCode: shouldUseWalletConnect
+    mobile: isMobile()
```

Comment:
I think that it would be better to separate the inline functions into: `getMobile`, `getQrCode`, and `getExtension` standalone functions.

It will improve readability.

**The expected end result:**:
```typescript
mobile: getMobile(),
qrCode: getQrCode(),
extension: getExtension()
```